pkg/plugin/conntrack/_cprog/conntrack.c

```diff
@@ -15,6 +16,20 @@ struct tcpmetadata {
 	__u32 tsecr; // TCP timestamp echo reply
 };
 
+struct conntrackmetadata {
```

Comment:
i think we discussed this offline, but do you think if it would be simpler/more practical if we just maintain the bytes and packet counts for the entire connection instead of its individual direction? cc @vakalapa

Reply:
@nddq I think it makes sense to keep both since it helps in analyzing the amount of data being sent and received, in a scenario where users need to understand network usage patterns and performance. This can also help  identifying and troubleshooting problematic connections. An example is detecting asymmetrical traffic patterns or security monitoring.